Sake/Mixer.cs

```diff
@@ -24,6 +26,7 @@ public Mixer(uint feeRate = 10, ulong minAllowedOutputAmount = 5000, uint inputS
             DenominationsPlusFees = CreateDenominationsPlusFees();
         }
 
+        public int AvailableVsize { get; } = 255;
```

Comment:
I don't know what this is and you put this magic number to 2 different places. Let's have it only on one place and have a good comment what this is: https://github.com/nopara73/Sake/pull/12/files#diff-fd24248c248385875c88dd6645de6e55088bbfd231b88775df1f0e5ceb0d0eeaR14

Reply:
I checked the actual logic. See my latest commit.